source/dynamic_data_attachment.md

```diff
@@ -0,0 +1,97 @@
+
+Users can now retrieve any information (such as _text_, _integer_, _decimal_, _select_one response_, _select_multiple response_, _GPS points_) to a child project (aka round 2 project) that has already been collected and stored in a parent project (aka round 1 project). However, media information (such as _image_, _audio_, _video_, _uploaded files_, etc.) cannot be retrieved at the moment.
+
+At the same time, users should also be able to perform various calculations such as counting responses entered and stored in a parent project directly through the child project. Similarly, users are also able to compute the _sum_, _maximum_, or _minimum_ value of a particular variable collected and stored in a parent project directly through a child project.
+
+## Linking a project dynamically through xlsform:
+
+While trying this approach through the xlsform, users will require two xlsform. The first xlsform (also known as <a download href="./_static/files/dynamic_data_attachment/Round_1.xlsx"> parent form or round 1 form</a>) is generally a normal xlsform that we often design. The second xlsform (also known as <a download href="./_static/files/dynamic_data_attachment/Round_2.xlsx"> child form or round 2 form</a>) is slightly different from the first xlsform, especially in two aspects, i.e., it has an xml-external question type, and it also has a calculation column that holds the path to the survey project being linked.
+
+Users could design their parent project as shown in the image below:
+
+![image](/images/dynamic_data_attachment/Round_1.png)
+
+In the same way, users could design their child project as shown in the image below:
+
+![image](/images/dynamic_data_attachment/Round_2.png)
+
+__Note:__ The name used for the _xml-external_ question type in the child project is crucial for linking the project with the parent project. In the above example, it was named survey. You can give it a different name as well _(it can only consist of Latin characters and numbers)_.
+
+## Understanding the syntax used under the calculation column of the child project:
+
+The _survey_ used in all of the syntaxes explained below is the name provided for the _xml-external_ question type as already outlined above. Hence, if you accidentally give a different name (other than _survey_) here, your calculation syntax should be invalid.
+
+__count(instance('survey')/root/data[P_Enumerator]):__ Will return users the total count of what was entered in P_Enumerator (parent project).
```

Comment:
Maybe have these set up in a table and use back-ticks for the code rather than bold, so `count(instance('survey')/root/data[P_Enumerator])` rather than __count(instance('survey')/root/data[P_Enumerator])__

You could do something like this:
XLSForm Syntax | Description
---|---
`count(instance('survey')/root/data[P_Enumerator])` | Total count of `P_Enumerator` in the _parent project_
`count(instance('survey')/root/data[P_Enumerator = current()/../C_Enumerator])` | Total count of `C_Enumerator` in the _child project_, matching/comparing with `P_Enumerator` in the _parent project_